tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/schema/dockerTaskEngineInput.ts

```diff
@@ -0,0 +1,128 @@
+import convict from 'convict';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+export class DockerTaskEngineInput {
+    configFilePath: string;
+    initOutput: string;
+    generateAndBuildInputJson: string;
+    generateAndBuildOutputJson: string;
+    mockTestInputJson: string;
+    mockTestOutputJson: string;
+    headSha: string | undefined;
+    headRef: string | undefined;
+    repoHttpsUrl: string;
+    serviceType: string;
+    mockServerHost?: string;
+    initTaskLog: string;
+    generateAndBuildTaskLog: string;
+    mockTestTaskLog: string;
+    taskResultJson: string;
+    changeOwner: boolean;
+}
+
+export const dockerTaskEngineConfig = convict<DockerTaskEngineInput>({
```

Comment:
dockerTaskEngineConfig -> dockerTaskEngineInput
others are both used Input instead of Config

Reply:
good comments. Forget to change it here.